ls8/cpu.py

```diff
@@ -58,39 +92,6 @@ def load(self, filename):
 
         address = 0
 
-        # open file
```

Comment:
I don't know if there is a reason you kept this old code but a good practice is to clean up your code by removing commented out code blocks and using simple 1 or 2 line comments for your code